src/lTerm_widget.mli

```diff
@@ -253,6 +256,31 @@ class ['a] radiobutton : 'a radiogroup -> string -> 'a -> object
 
 end
 
+(** {6 Scrollbars} *)
+
+(** methods required for a widget to supprt scrollbars *)
+class type scrollable = object
+  method full_size : LTerm_geom.size
+    (* full size of scrollable window *)
+
+  method offset : LTerm_geom.coord
```

Comment:
Yep, this way it would be more natural to use a scrollbar for something that is not a 2d position